src/lib.rs

```diff
@@ -932,13 +932,13 @@ impl<'a, T: 'a + CellType> DoubleEndedIterator for Rows<'a, T> {
 impl<'a, T: 'a + CellType> ExactSizeIterator for Rows<'a, T> {}
 
 /// Struct with the key elements of a table
-pub struct Table<T> {
+pub struct Table<T: CellType> {
```

Comment:
I think it is better to refrain from adding constraints on the structs, the impl are enough, more generic and more future proof.